genref/README.md

```diff
@@ -0,0 +1,74 @@
+# API Reference generator
+
+This is an API reference docs generator that uses the
+[k8s.io/gengo](https://godoc.org/k8s.io/gengo) project to parse types and
+generate API documentation from it.
+
+This tool relies only on the Go source code containing the type definitions.
+For references to external type definitions, especially those from the
+`k8s.io/kubernetes` repo, you can tell the tool to generate links to the
+existing docs, which can be the GoDocs if no better option exists.
+
+## Try it out
+
+1. Clone this repository.
+
+2. Make sure you have go1.15+ installed. Run `make` to build the `genref`
+   binary:
+
+   ```
+   cd genref
```

Comment:
Does it make sense to build this tool along with the other generators?

Reply:
Yes. We will need to update the release workflow a little bit to include the artifacts generated here.